sequence_processing_pipeline/tests/test_QCJob.py

```diff
@@ -685,27 +690,17 @@ def test_command_chaining(self):
                 lines_obs = [x.strip() for x in lines_obs]
                 if remove_this:
                     lines_obs = [x.replace(remove_this, '') for x in lines_obs]
-                    if lines_obs[0] != config['first_line']:
-                        print("FIRST LINE OBSERVED ##################")
-                        print(lines_obs[0])
-                        print("FIRST LINE EXPECTED ##################")
-                        print(config['first_line'])
-                        print("##################")
-                    if lines_obs[-1] != config['last_line']:
-                        print("LAST LINE OBSERVED ##################")
-                        print(lines_obs[-1])
-                        print("LAST LINE EXPECTED ##################")
-                        print(config['last_line'])
-                        print("##################")
+                print("LAST LINE: %s" % lines_obs[-1])
```

Comment:
I think is fine to also remove the last print, no?

Reply:
definitely. done!